Bracket.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+
+/**
+ * Bracket Class
+ * Created by Matt and Dan on 5/1/2017.
+ */
+public class Bracket {
+    //Attributes
+    ArrayList<String> bracket;
+    String name;
+    static final int EAST_BRACKET = 63;
+    static final int WEST_BRACKET = 79;
+    static final int NORTH_BRACKET = 95;
+    static final int SOUTH_BRACKET = 111;
+
+    //Constructor
+    /**
+     *Cosntructor using an ArrayList of strings to start
+     */
+    public Bracket(ArrayList<String> starting){
+        bracket = starting;

```

Comment:
Yeah, that's exactly it.
```
bracket = starting;
```
Will make bracket reference the same array as starting, meaning any changes to the bracket array in this object will change the bracket in the starting array.
```
bracket = new ArrayList<String>(starting);
```
uses the ArrayList constructor to perform a deep copy and create a new array initialized to the values in starting, meaning we can make changes to it without impacting starting, which is what we want.